packages/atomic/src/components/common/atomic-focus-trap.tsx

```diff
@@ -96,11 +129,15 @@ export class AtomicFocusTrap {
     getFirstFocusableDescendant(this.host)?.focus();
   }
 
-  connectedCallback() {
-    this.activeChanged(this.active);
+  async connectedCallback() {
```

Comment:
this could technically block the connectedCallback of the parent `atomic-search-interface` I think, can you use componentWillLoad?

Reply:
I will avoid awaiting anything in `connectedCallback` then.